app/src/App.tsx

```diff
@@ -1,57 +1,19 @@
 import React from 'react';
-import logo from './logo.svg';
-import Counter from './features/counter/Counter';
 import './App.css';
+import Home from './features/home/home';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Reference from './features/reference/reference';
+import CreateSurvey from './features/createSurvey/createSurvey';
 
 const App = function () {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <Counter />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <span>
-          <span>Learn </span>
-          <a
-            className="App-link"
-            href="https://reactjs.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            React
-          </a>
-          <span>, </span>
-          <a
-            className="App-link"
-            href="https://redux.js.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Redux
-          </a>
-          <span>, </span>
-          <a
-            className="App-link"
-            href="https://redux-toolkit.js.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Redux Toolkit
-          </a>
-          ,<span> and </span>
-          <a
-            className="App-link"
-            href="https://react-redux.js.org/"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            React Redux
-          </a>
-        </span>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="reference" element={<Reference />} />
+        <Route path="createSurvey" element={<CreateSurvey />} />
```

Comment:
url 명명은 camel case 대신 보통 dash 로 구분하니 시간 되시면 바꾸어 주세요!

```suggestion
        <Route path="create-survey" element={<CreateSurvey />} />
```